src/zui/ZUIScrollingContainer/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+import { makeStyles } from '@mui/styles';
+import { Box, BoxProps, Theme } from '@mui/material';
+
+type ZUIScrollingContainerProps = Omit<BoxProps, 'className' | 'overflow'> & {
+  disableHorizontal?: boolean;
+};
+
+const useStyles = makeStyles<Theme, ZUIScrollingContainerProps>({
+  container: {
+    overflowX: (props) => (props.disableHorizontal ? 'auto' : 'scroll'),
+    overflowY: 'scroll',
+  },
+});
+
+const ZUIScrollingContainer: FC<ZUIScrollingContainerProps> = ({
+  children,
```

Comment:
Maybe i should refactor the choice gallery to use this component...

Reply:
Not urgently, but maybe at some point!